test/functional/system/blocks/chain.delete_last_block.js

```diff
@@ -423,7 +423,7 @@ describe('system test (blocks) - chain/deleteLastBlock', () => {
 					localCommon.addTransactionsAndForge(library, [], done);
 				});
 
-				it('should validate account data from sender', done => {
+				it('should validate account data from sender after forging a block with pool transaction', done => {
```

Comment:
`should validate account data from sender after forging a block with transaction pool`